integration/spark-common/src/main/scala/org/apache/spark/sql/catalyst/CarbonDDLSqlParser.scala

```diff
@@ -218,6 +221,45 @@ abstract class CarbonDDLSqlParser extends AbstractCarbonSparkSQLParser {
     )
   }
 
+  def getField(col: StructField): Field = {
```

Comment:
Not sure what is the purpose of this func

Reply:
This is existing base code. It is moved from `getFields` method of CarbonSpark2SqlParser.scala and made it as reusable method. So that it can be called for newly created column as well. It is done to avoid code duplication.